src/pages/index.css

```diff
@@ -0,0 +1,4 @@
+html, body, #gatsby-focus-wrapper, #___gatsby {
+	height: 100%;
+	width: 100%;
+}
```

Comment:
Do we really need this extra CSS? Is styling the homepage container not enough? Also why does this style those gatsby IDs?

Reply:
Yeah. Those are the IDs of two divs gatsby puts around every page, so even if we have a homepage container with styling of height: 100% , the container takes 100% of the height the wrappers allow it to which is not 100% of the page.